addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -0,0 +1,116 @@
+            <t t-set="address">
+                <t t-out="address"/>
+                <div class="mb-3"/>
+            </t>
+        </xpath>
+
+        <!--
+            INFORMATIONS BLOCK
+        -->
+
+        <!-- Billing Address -->
+        <xpath expr="(//div[@id='informations']/*)[last()]" position="after">
+            <t t-if="o._is_company_french() and o.partner_invoice_id">
+                <t t-set="company_address">
+                    <address t-field="o.partner_id" class="m-0"
+                        t-options='{"widget": "contact", "fields": ["address"], "no_marker": True}'/>
+                </t>
+
+                <t t-set="billing_address">
+                    <address t-field="o.partner_invoice_id" class="m-0"
+                        t-options='{"widget": "contact", "fields": ["address"], "no_marker": True}'/>
+                </t>
+
+                <t t-if="company_address != billing_address">
+                    <div class="col-auto col-3 mw-100 mb-2">
+                        <strong>Adresse de facturation&#160;:</strong> <!-- EN: Billing address -->
+                        <p class="m-0" t-out="billing_address"/>
+                    </div>
+                </t>
+            </t>
+        </xpath>
+
+        <!-- Delivery address -->
+        <xpath expr="(//div[@id='informations']/*)[last()]" position="after">
+            <t t-if="o._is_company_french() and o.partner_shipping_id">
+                <t t-set="billing_address">
+                    <address t-field="o.partner_invoice_id" class="m-0"
+                        t-options='{"widget": "contact", "fields": ["address"], "no_marker": True}'/>
+                </t>
+
+                <t t-set="delivery_address">
+                    <address t-field="o.partner_shipping_id" class="m-0"
+                        t-options='{"widget": "contact", "fields": ["address"], "no_marker": True}'/>
+                </t>
+
+                <t t-if="billing_address != delivery_address">
+                    <div class="col-auto col-3 mw-100 mb-2">
+                        <strong>Adresse de livraison&#160;:</strong> <!-- EN: Delivery address -->
+                        <p class="m-0" t-out="delivery_address"/>
+                    </div>
+                </t>
+            </t>
+        </xpath>
+
+        <!-- Product types -->
+        <xpath expr="(//div[@id='informations']/*)[last()]" position="after">
+            <t t-if="o._is_company_french()">
+                <t t-set="tax_scopes" t-value="o.invoice_line_ids.mapped('tax_ids.tax_scope')"/>
+                <t t-set="has_service" t-value="'service' in tax_scopes"/>
+                <t t-set="has_consu" t-value="'consu' in tax_scopes"/>
+
+                <t t-if="has_service or has_consu">
+                    <div class="col-auto col-3 mw-100 mb-2">
+                        <strong>Type d'opération&#160;:</strong> <!-- EN: Operation type -->
```

Comment:
same comment for the use of strong here + must be in english and no need for the space in html